crates/swc_ecma_minifier/tests/terser/compress/collapse_vars/collapse_vars_export/input.js

```diff
@@ -0,0 +1,4 @@
+export var a;
```

Comment:
Please move this vars to `tests/compress/fixture/issues/3709`.

Reply:
`terser` is a directory for tests ported from terser

Reply:
https://github.com/swc-project/swc/tree/1fbe209c53df8f6ac437e3ec53e2d8879131481c/crates/swc_ecma_minifier/tests/compress/fixture/issues